server/main_server/src/rodsServer.cpp

```diff
@@ -1131,7 +1161,7 @@ int serverMain(const bool enable_test_mode = false, const bool write_to_stdout =
                 }
 
                 using namespace std::chrono_literals;
-                std::this_thread::sleep_for(100ms);
+                std::this_thread::sleep_for(10ms);
```

Comment:
What is the motivation behind this change?

Reply:
This was a hack before I got around to writing the spinlock. It also happens to negate the single benefit of the spinlock that I had in mind anyway, the latency, so I will replace the spinlock with a barrier.